### DISCUSSION TITLE
Led indexing

### DISCUSSION THREAD

**Discussion by kozmaz87** [PlummersSoftwareLLC/NightDriverStrip#107] [Q&A]
I have given the demo env a go. The good news is that it more or less worked out of the box on Win10 x64 on a Ryzen processor. I connected a logic analyzer with WS2812b decoder and it says that even though I told it to do 14 LEDs it counts up the index to 2B. Is this intentional? The initialization says 14 LEDs 
![image](https://user-images.githubusercontent.com/4070837/146413731-905a8c18-cfe4-4ef5-b883-4f2e9bd950a1.png)

but the logic analyzer says... no mate it is addressing to 2B. This does not cause issues on a single strip but I would imagine break matrix arrangements... or I am missing something... 

![image](https://user-images.githubusercontent.com/4070837/146414052-df14ac14-b123-44ac-866d-91882235ea7d.png)

(Noteworthy to mention it takes clicking build twice by the time the project successfully builds for some reason for me anyway. Clean + Build 1 fails on a missing file + Build 2 succeeds.)

**Comment by robertlipe** [comment]
I come from the future. I fixed a bug in the LED counting code just a few days ago that would have almost explained this. The overcount doesn't quite match, but I didn't roll the code back to 2011 to compare. I *think* the trunk as of 8/2023 has this fixed. No reason to spend our time budget wiggling wires that are just going to be shifted right off the end of the wire, right?

Your analyzer is very cool. I haven't seen one that decoded WS281x.
